python/lsst/ts/xml/generate_csv_table.py

```diff
@@ -32,7 +32,7 @@ def generate_csv_table():
     tree = ElementTree.parse(sal_interfaces_dir / "SALSubsystems.xml")
     root = tree.getroot()
 
-    outpath = utils.get_pkg_root() / "doc" / "subsystemData.csv"
+    outpath = utils.get_pkg_root() / "subsystemData.csv"
```

Comment:
I still very strongly think that the `subsystemData.csv` file should not be moved from the `doc` directory. Are you sure that this is absolutely necessary? I don't think that any changes to the `generate_csv_table.py` file are necessary, including the change to line 141.